docs/src/pages/[platform]/components/link/LinkPropControls.tsx

```diff
@@ -20,19 +27,29 @@ export const LinkPropControls: LinkPropControlsInterface = ({
   setIsExternal,
   textDecoration,
   setTextDecoration,
+  children,
+  setChildren,
 }) => {
   return (
     <Flex direction="column">
+      <TextField
+        value={children as string}
+        onChange={(event) => setChildren(event.target.value)}
+        label="body"
```

Comment:
Why label this as `body` when it's demonstrating the `children` prop?

Reply:
I was following the pattern from the [Alert docs](https://ui.docs.amplify.aws/react/components/alert). But I agree `children` makes more sense, I'll change it